config/deploy/dev.rb

```diff
@@ -0,0 +1 @@
+server 'upaya-dev.18f.gov', roles: %w(web app db)
```

Comment:
Huh. It didn't work for me. I got a Net:SSH error. When I added the user, it worked.